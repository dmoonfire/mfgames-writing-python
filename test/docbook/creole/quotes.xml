<?xml version="1.0" encoding="UTF-8"?><article xmlns='http://docbook.org/ns/docbook' version="5.0"><info><title>Quoted Example</title></info><simpara>This is an example that has <quote>many quotes</quote> of note.</simpara></article>
