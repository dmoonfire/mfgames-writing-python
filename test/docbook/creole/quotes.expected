= Quoted Example

This is an example that has "many quotes" of note.
